proposals-CG-WG.md

```diff
@@ -29,70 +29,73 @@ The purpose of Stage 0 proposals is to allow anyone to raise and explore the Pro
 The purpose of Stage 1 proposals is to explore the Solution Space and pick a preferred direction. This is where most of the work gets done because it involves exploring alternatives, understanding tradeoffs, gathering implementation experience, incubating alternatives, gathering evidence of demand and fitness for purpose, and finally, identifying the best out of the many alternatives.
 
   * What's needed from individuals?
-    * [ ] Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md)
-    * [ ] An understanding of the Problem Space
-    * [ ] Optionally, alternatives under consideration
+    * [ ] Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md).
+    * [ ] An understanding of the Problem Space.
+    * [ ] Optionally, alternatives under consideration.
   * What's asked of the **Community Group**?
-    * [ ] The Community Group consensus that the problem is worth spending the Community Group’s time working on
+    * [ ] The Community Group consensus that the problem is worth spending the Community Group’s time working on.
     * [ ] A home for incubating the proposal. Small features will incubate in issues. If and when the champions are ready to more thoroughly document their proposal, the WG chairs will create a repo for the champions to develop the feature (e.g., [example](https://github.com/fedidcg/LightweightFedCM)). 
 
 # Stage 2: Formalization
 
-The purpose of Stage 2 Proposals is to formally specify the best (and seemingly workable) alternative that was identified in the prior step: handle corner cases, integrate with other parts, reconcile with other proposals, and resolve the concerns identified at the entrance. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.
+The goal of Stage 2 is to refine the preferred solution into a detailed, cohesive proposal that addresses known issues and integrates feedback from stakeholders. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.  This stage focuses on preparing the proposal for Working Group review and creating a complete, formal draft.
 
   * What's needed from champions?
-    * [ ] An [explainer](https://tag.w3.org/explainers/)
-    * [ ] Alternatives and trade-offs considered
-    * [ ] A specific preferred proposal, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)
-    * [ ] Browser and developer implementation experience (e.g., a prototype, dev trials, origin trials, etc) 
-    * [ ] Confidence of developer demand and fitness for purpose (e.g. a developer that needs this proposal)
+    * [ ] An [explainer](https://tag.w3.org/explainers/).
```

Comment:
Can I suggest that we keep these editorial fixes outside of this PR so that we can focus on the normative changes? For example, I almost dismissed the change below because I assumed that it was just making editorial changes rather than normative.

The editorial changes are perfectly valid, I just think that they may be best done in isolation rather than together with this spec PR.